src/main/java/uk/gov/hmcts/reform/civil/utils/HearingUtils.java

```diff
@@ -54,4 +59,55 @@ public static BigDecimal getFastTrackFee(int claimFee) {
             return new BigDecimal(34600);
         }
     }
+
+    public static String getHearingType(CaseData caseData) {
+        switch (caseData.getChannel()) {
+            case IN_PERSON:
+                return caseData.getHearingLocation().getValue().getLabel();
+            case VIDEO:
+                return "videoconference";
+            case TELEPHONE:
+                return "telephone";
+            default:
+                return "not defined";
+        }
+    }
+
+    public static String getHearingDuration(CaseData caseData) {
+        switch (caseData.getHearingDuration()) {
+            case MINUTES_30:
+                return "30 minutes";
+            case MINUTES_60:
+                return "1 hour";
+            case MINUTES_90:
+                return "1 and half hours";
+            case MINUTES_120:
+                return "2 hours";
+            case MINUTES_150:
+                return "2 and half hours";
+            case MINUTES_180:
+                return "3 hours";
+            case MINUTES_240:
+                return "4 hours";
+            case DAY_1:
+                return "1 day";
+            case DAY_2:
+                return "2 days";
+            default:
+                return "not defined";
+        }
+    }
+
+    public static String formatHearingFee(Fee hearingFee) {
+        if (nonNull(hearingFee) && hearingFee.getCalculatedAmountInPence().intValue() > 0) {
+            StringBuilder builder = new StringBuilder(hearingFee.getCalculatedAmountInPence().toString());
+            builder.delete(builder.length() - 2, builder.length());
+            if (builder.length() > 3) {
+                builder.insert(1, ',');
+            }
+            builder.insert(0, '£');
+            return builder.toString();
+        }
+        return "";
```

Comment:
Why not using `DecimalFormat` to do the work for you?

Something like:
```
DecimalFormat formatter = new DecimalFormat("#,###.00");
formatter.format(value);
```

Also, if I'm reading the code correctly we're truncating the decimals. Are fees expected to always be whole pound values?

Reply:
Fee values are fixed and do not contain decimals.